sass/blocks/form/_form.scss

```diff
@@ -0,0 +1,324 @@
+.form {
+  &__section {
+    &::after {
+      content: "";
+      display: block;
+      border-top: 1px solid $form-divider;
+    }
+  }
+
+  &__inner {
+    position: relative;
+    padding: 1.8rem 6.25% 0.8rem;
+
+    &--spacing-bottom {
+      padding-bottom: 19px;
+    }
+  }
+
+  &__field {
+    display: block;
+    width: 100%;
+
+    &--flexgrid {
+      display: flex;
+      align-items: stretch;
+      padding-top: 0.4rem;
+      padding-bottom: 1.6rem;
+    }
+
+    &--phone {
+      padding-top: 0;
+
+      &::before {
+        content: "";
+        position: absolute;
+        top: 9.7rem;
+        left: 24.3rem;
```

Comment:
you'd better use right: 0 here
![image](https://user-images.githubusercontent.com/1570578/34539937-876084da-f0ca-11e7-9dc0-8fe6b63fdda1.png)

Reply:
also looks like your input is not stretching to 100% and centring. I think that should be a behaviour.
Looking into your code I see you're struggling with width, I would switch to box-sizing: border-box; for all the inputs, it should be more nice, other than having 77% of weight have 100.